hamp_radar/postprocess.py

```diff
@@ -0,0 +1,6 @@
+import xarray as xr
+from decoders import decode_time
```

Comment:
Maybe `decode_time` shouldn't be a decoder but rather a postprocessing function?

Reply:
Yes I agree! I'll move it there :)

Reply:
(was hoping you'd say that ;) )